editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -685,7 +697,10 @@ function createBlanklineTracker(): (statement: TS.Node, printedCode: string) =>
   let lastLineWasVariable = false
   let lastLineWasMultiLineStatement = false
 
-  return (statement: TS.Node, printedCode: string): boolean => {
+  return (statement: TS.Node, rawPrintedCode: string): boolean => {
+    // Awful hack until we implement #851
+    const printedCode = replaceAll(rawPrintedCode, '/** @jsx jsx */', '').trim()
```

Comment:
what behavior are we altering here? is this to prevent a newline below the pragma? or was this tripping up the test cases and throwing an error?

Reply:
Yeah this is because of the blank line insertion implemented in #840 that is about to get binned anyway as part of #851 (which I'm working on right now). It was mainly because this was tripping test cases that weren't expecting the extra blank line it was adding into the code.